custom-elements.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Why not exclude this generated file (it's regenerated anyway on prestart)? We used to commit analysis.json files earlier but as maintaining it turned out to be inefficient.

Reply:
Done.